src/terra-dev-site/IssueForm/BugForm.jsx

```diff
@@ -0,0 +1,155 @@
+            isInvalid={meta.invalid && meta.touched}
+            help="Please specify the exact steps you took for this bug to occur. Provide as much detail as possible so we're able to reproduce these steps."
+            maxWidth="50em"
+            onChange={(event) => { input.onChange(event.target.value); setSteps(event.target.value); }}
+            value={input.value}
+            inputAttrs={{ ...input }}
+          />
+        )}
+      </Field>
+      <Field
+        name="expected"
+        validate={validateForm}
+      >
+        {({ input, meta }) => (
+          <TextareaField
+            label="Expected Behavior"
+            aria-label="A clear and concise description of what you expected to happen."
+            inputId="description"
+            required
+            error={meta.error}
+            isInvalid={meta.invalid && meta.touched}
+            help="A clear and concise description of what you expected to happen."
+            maxWidth="50em"
+            onChange={(event) => { input.onChange(event.target.value); setExpected(event.target.value); }}
+            value={input.value}
+            inputAttrs={{ ...input }}
+          />
+        )}
+      </Field>
+      <Field
+        name="environment"
+        validate={validateForm}
+      >
+        {({ input, meta }) => (
+          <TextareaField
+            label="Environment"
+            aria-label="Include as many relevant details about the environment you experienced the bug in."
+            inputId="environment"
+            required
+            error={meta.error}
+            isInvalid={meta.invalid && meta.touched}
+            help="Include as many relevant details about the environment you experienced the bug in."
+            maxWidth="50em"
+            value={!input.value ? environmentTemplate : input.value}
+            onChange={(event) => { input.onChange(event.target.value); setEnvironment(event.target.value); }}
+            inputAttrs={{ ...input }}
+          />
+        )}
+      </Field>
+      <TextareaField
+        label="Additional Context / Screenshots"
+        aria-label="Add any other context about the feature here. If applicable, add screenshots to help explain."
+        inputId="context"
+        help="Add any other context about the feature here. If applicable, add screenshots to help explain."
+        maxWidth="50em"
+        value={context}
+        onChange={event => setContext(event.target.value)}
+      />
+      <TextareaField
+        label="Possible Solution"
+        aria-label="If you have suggestions to fix the bug, let us know."
+        inputId="solution"
+        help="If you have suggestions to fix the bug, let us know."
+        maxWidth="50em"
+        value={solution}
+        onChange={event => setSolution(event.target.value)}
+      />
+      <TextareaField
+        label="Mentions"
+        aria-label="@ Mention anyone on the terra team that you have been working with so far."
```

Comment:
Let's remove the `@` character from here. GitHub issue form has a feature to auto-suggest usernames when you type `@` on the form. I'm not too interested in replicating that here and think most people won't know our GitHub usernames so it would be eaiser to understand if they just type in our names here.

Reply:
Updated: [d658958](https://github.com/cerner/terra-ui/pull/151/commits/d6589587b241e85489f9f156c29d1dab408f90d0)